### PR TITLE
Implement DA main page

### PR DESCRIPTION
**Pull request by torztomasz**

### CODE REVIEW

File: packages/frontend2/types.d.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import '@tanstack/react-table' //or vue, svelte, solid, qwik, etc.
```
Comment (imxeno) [inline packages/frontend2/types.d.ts:1]: remove comment

File: packages/frontend2/src/app/_components/table/table-faceted-filter.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+          <div className="flex size-3 items-center justify-center rounded-sm bg-black dark:bg-white">
+            <CloseIcon className="size-2.5 fill-white dark:fill-black dark:group-hover:fill-gray-950" />
+          </div>
+        </div>
+      </button>
+    )
+  }
+
+  const showDrawer = breakpoint === 'tablet' || breakpoint === 'mobile'
+  return showDrawer ? (
```
Comment (imxeno) [inline packages/frontend2/src/app/_components/table/table-faceted-filter.tsx:60]: It's very hard to see where the alternative case of this ternary operator starts. Let's maybe do the same as above, so:
```ts
 if(showDrawer) {
   return ...
 }
 
 return ...
 ```
 It's way more readable.

File: packages/frontend2/src/app/(new)/data-availability/summary/_components/table/da-bridge-cell.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { type DataAvailabilityProvider } from './columns'
+
+interface Props {
+  daBridge: DataAvailabilityProvider['daBridge']
+}
+export function DaBridgeCell({ daBridge }: Props) {
+  if (!daBridge) return 'No bridge'
```
Comment (imxeno) [inline packages/frontend2/src/app/(new)/data-availability/summary/_components/table/da-bridge-cell.tsx:8]: Should this really return plain string?

**Review by imxeno — APPROVED** [review]

File: packages/frontend2/src/app/(new)/data-availability/summary/_components/table/da-bridge-cell.tsx (hunk 3 above)
Comment (torztomasz) [reply packages/frontend2/src/app/(new)/data-availability/summary/_components/table/da-bridge-cell.tsx:8]: yup, no need to wrap it in anything

**Pull request merged**